code/game/objects/items/stacks/seaweed.dm

```diff
@@ -0,0 +1,13 @@
+/obj/item/stack/seaweed
+	name = "seaweed sheet"
+	desc = "A sheet of seaweed."
```

Comment:
Maybe a better description? Ask people on discord for ideas.

Reply:
new one good?

Reply:
Yeah, looks better